addons/pos_online_payment/models/pos_payment_method.py

```diff
@@ -114,5 +114,5 @@ def _get_or_create_online_payment_method(self, company_id, pos_config_id):
                     'company_id': company_id,
                 })
                 if not payment_method_id:
-                    raise ValidationError('Could not create an online payment method (company_id=%d, pos_config_id=%d)' % (company_id, pos_config_id))
+                    raise ValidationError(_('Could not create an online payment method (company_id=%d, pos_config_id=%d)' % company_id, pos_config_id))
```

Comment:
you'll need to get rid of the "%" operator and replace it with comma ",". From how I understand it, the translation function has a built in string interpolation.

Reply:
updated the pr